src/chainparamsseeds.h

```diff
@@ -1,5 +1,10 @@
 #ifndef BITCOIN_CHAINPARAMSSEEDS_H
 #define BITCOIN_CHAINPARAMSSEEDS_H
+/**
```

Comment:
remove these lines 3 - 7

Reply:
done sir. Thank you again for pointing me in the right direction